### ISSUE TITLE
Should we split the mainfunctionality into separate gems?

### ISSUE THREAD

**Issue by polarblau** [polarblau/spinaltap#2]
**Main functionality:**
- Generators
- Validation (using Rails' validations in Backbone)
- Routing (using Rails' validations in Backbone)
- ???

gems: `vertebrae, vertebrae-validations, vertebrae-routing, …`

**Comment by pietia** [comment]
That would be robust solution. The question is what to put into the
base gem which should be fully functional for basic usage.

**Comment by polarblau** [comment]
Yep, I've been wondering the same? Maybe the generators for now, extracting it later as well if necessary?

**Comment by pietia** [comment]
+1
